tests/worksheet_test.py

```diff
@@ -1440,6 +1440,13 @@ def test_batch_get(self):
         self.assertEqual(value_ranges[1].range, "Sheet1!B4:D4")
         self.assertEqual(value_ranges[0].first(), "A1")
 
+    @pytest.mark.vcr()
+    def test_cell_return_first(self):
+        cell = self.sheet.cell(1, 1)
+
+        self.assertIsInstance(cell, gspread.cell.Cell)
+        self.assertIsNone(cell.value)
```

Comment:
looks good. does cell sometimes return `""` instead of `None`? Is that what those guys in the linked issue were saying?

Reply:
I don't remember cells returning `""` empty string. So far they return either a value or `None` (of course that goes without the use of default value etc).

And no that's not what they were speaking of, their issue is related to us returning the string `"None"` instead of the value `None`. 

But yes the test here is to ensure what they were saying if you get an empty cell you get a `None` value.